src/utils.js

```diff
@@ -53,8 +53,12 @@ exports.logHelper = (...args) => {
         return
     }
 
-    const line = args[0].slice(args[0].indexOf(sendString) + sendString.length)
-    const command = JSON.parse(line)
+    // It extracts sent command in the args[0] like
+    // 2022-06-03T00:16:49.613Z puppeteer:protocol:SEND ► [
+    //  '{"sessionId":"ea27a779-42d2-4c8c-9dac-10634741a60e","method":"Runtime.callFunctionOn"}'
+    //]
+    const line = args[0].slice(args[0].indexOf(sendString) + sendString.length).replace(/\n/g, '').trim()
```

Comment:
Since the command is now surrounded by `[]`, does that imply this is now a list and there could be multiple commands?

Reply:
I'm not quite sure. Let me have a check

Reply:
checked. It's a list but only contains one item. Should be safe.

Reply:
> checked. It's a list but only contains one item. Should be safe.

And that's not just because the test was doing only one action?

Reply:
yup. only one action.